MonstieWash/Assets/Scripts/Items/ItemLinker.cs

```diff
@@ -0,0 +1,139 @@
+                if (taskScript != null)
+                {
+                    m_taskScript.Add(taskScript);
+                }
+                else
+                {
+                    Debug.LogError($"ItemLinker on {name} is setup incorrectly. Could not find a task script on child {child.name}!");
+                }
+            }
+        }
+        else
+        {
+            taskScript = GetComponent<ITaskScript>();
+            if (taskScript != null)
+            {
+                m_taskScript.Add(taskScript);
+            }
+            else
+            {
+                Debug.LogError($"ItemLinker on {name} is setup incorrectly. Could not find a task script on {name}!");
+            }
+        }
+    }
+
+/// <summary>
+/// Saves data from linked item in scene to link object
+/// </summary>
+    private void SaveItem()
+    {
+        linkObject.Reset();
+        if (link == LinkMethod.Group)
+        {
+            foreach (Transform child in transform)
+            {
+                linkObject.positions.Add(FlipItemPos(child));
+                linkObject.rotations.Add(FlipItemRot(child));
+                linkObject.itemData.Add(new List<object>(child.GetComponent<ITaskScript>().SaveData()));
+            }
+        }
+        else
+        {
+            linkObject.positions.Add(FlipItemPos(transform));
+            linkObject.rotations.Add(FlipItemRot(transform));
+            linkObject.itemData.Add(new List<object>(m_taskScript[0].SaveData()));
+        }
+        linkObject.loadOnEnable = true;
+    }
+
+/// <summary>
+/// Loads data from link object to linked item in scene
+/// </summary>
+    private void LoadItem()
+    {
+        if (linkObject.loadOnEnable == true) 
+        {
+            if (link == LinkMethod.Group)
+            {
+                for (int i = 0; i < transform.childCount; i++)
+                {
+                    transform.GetChild(i).position = linkObject.positions[i];
+                    transform.GetChild(i).rotation = linkObject.rotations[i];
+                    transform.GetChild(i).GetComponent<ITaskScript>().LoadData(linkObject.itemData[i]);
+                }
+            }
+            else
+            {
+                transform.position = linkObject.positions[0];
+                transform.rotation = linkObject.rotations[0];
+                m_taskScript[0].LoadData(linkObject.itemData[0]);
+            }
+            linkObject.loadOnEnable = false;
+        }
+    }
+
+    private void OnEnable()
+    {
+        m_gameSceneManager.OnSceneSwitch += SaveItem;
+        m_gameSceneManager.OnSceneChanged += LoadItem;
+        m_gameSceneManager.OnLevelEnd += CleanUp;
+    }
+
+    private void OnDisable()
+    {
+        m_gameSceneManager.OnSceneSwitch -= SaveItem;
+        m_gameSceneManager.OnSceneChanged -= LoadItem;
+        m_gameSceneManager.OnLevelEnd -= CleanUp;
+    }
+
+    private void OnApplicationQuit()
+    {
+        CleanUp();
+    }
```

Comment:
Just these methods should be moved up underneath `Awake()` to keep it consistent with the ordering of all the other scripts' methods.